### PR TITLE
Approved reviewers and permissions to handbook process

### PR DESCRIPTION
**Pull request by jasonblais**
@michaelschiffmm This is a proposal for approved reviewers and permission levels. Would be interested to hear your thoughts for BizOps.

### CODE REVIEW

File: company/how-to-guides-for-staff/how-to-update-handbook.md [hunk 1]
```diff
@@ -7,42 +7,27 @@
...
 
 ### Editing an Existing Page
 
-After you have opened the page you want to edit in GitHub, click the **Edit** pencil icon in the top right corner.
-
-This opens a Markdown formatted document of the page. To learn more about Markdown formatting, see the [Mattermost guide for formatting text](https://docs.mattermost.com/help/messaging/formatting-text.html), or [the guide from GitBook](https://docs.gitbook.com/editing-content/markdown).
-
-### Creating a Pull Request
+1. The quickest way to begin is navigating to the page you want to edit in [Mattermost Handbook](https://handbook.mattermost.com/), then clicking the **Edit on GitHub** icon in the top right navigation. This opens the page in GitHub that you can edit.
+2. Then, click the **Edit** pencil icon in the top right corner. This opens a Markdown formatted document of the page. To learn more about Markdown formatting, see the [Mattermost guide for formatting text](https://docs.mattermost.com/help/messaging/formatting-text.html), or [the guide from GitBook](https://docs.gitbook.com/editing-content/markdown).
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-update-handbook.md]:
```suggestion
2. In GitHub click the pencil icon in the navigation bar (above the page header) called **Edit the file in your fork of this project** to open the editable Markdown format page. 
- To learn more about Markdown formatting, see the [Mattermost guide for formatting text](https://docs.mattermost.com/help/messaging/formatting-text.html), or [the guide from GitBook](https://docs.gitbook.com/editing-content/markdown).
```

File: company/how-to-guides-for-staff/how-to-update-handbook.md [hunk 2]
```diff
@@ -7,42 +7,27 @@
...
+3. Make your edits. When you're ready to submit your changes, scroll to the bottom of the page to commit your changes.
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-update-handbook.md]:
```suggestion
3. Make your edits. When you're ready to submit your changes, scroll to the bottom of the page to commit your changes and start a pull request.
```

File: company/how-to-guides-for-staff/how-to-update-handbook.md [hunk 3]
```diff
@@ -7,42 +7,27 @@
...
 
-When you're ready to submit your changes, scroll to the bottom of the page to commit your changes.
-
-Add a descriptive title and comments to summarize the changes made, then select **Create a new branch for this commit and start a pull request**.
-
-After this, click the **Propose file change** button.
+* Add a descriptive title and comments to summarize the changes made, then select **Create a new branch for this commit and start a pull request**.
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-update-handbook.md]:
```suggestion
* Add a descriptive title if the default title isn't sufficient. Add an extended description to summarize the changes you've made. 
```

File: company/how-to-guides-for-staff/how-to-update-handbook.md [hunk 4]
```diff
@@ -7,42 +7,27 @@
...
+* Click the **Propose file change** button.
 
 ![](../../.gitbook/assets/how-to-update-handbook-commit-changes.png)
 
-### Opening a Pull Request
-
-On the next page, scroll down to compare changes with the original document, request **Reviewers** on the top right navigation, select relevant [labels](https://developers.mattermost.com/contribute/getting-started/labels/) for the pull request, then select **Create pull request**.
+4. On the next page, scroll down to compare changes with the original document, request **Reviewers** on the top right navigation, select relevant [labels](https://developers.mattermost.com/contribute/getting-started/labels/) for the pull request, then select **Create pull request**.
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-update-handbook.md]:
```suggestion
4. On the next page, scroll down to compare changes with the original document and then select **Create pull request**. 

5. Confirm that the title and description are correct. Then select **Create pull request**.  
```

File: company/how-to-guides-for-staff/how-to-update-handbook.md [hunk 5]
```diff
@@ -7,42 +7,27 @@
...
 
 ![](../../.gitbook/assets/how-to-update-handbook-create-new-pull-request.png)
 
-### Commenting on a Pull Request
-
-Once a pull request is submitted, multiple committers may comment on it and provide edits or suggestions which you can commit directly. You can also add line comments. Take a look at [Commenting on pull requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request) for more details.
-
-### Reviewing a Pull Request
-
 Once a pull request has been submitted and the correct labels assigned, the review process begins. This includes aligning the content with the Style Guide, validating processes, and tagging any other relevant committers.
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-update-handbook.md]:
```suggestion
Once a pull request has been submitted, a core committer with write-access assigns relevant reviewers and labels to kick off the review process.  The review process includes aligning the content with the Style Guide, validating the changes, and tagging any other relevant committers.
```

File: company/how-to-guides-for-staff/how-to-update-handbook.md [hunk 6]
```diff
@@ -79,3 +64,23 @@ Follow these two steps:
...
+5. @natalie-hub, @HilaryClarke: Signs off on changes to [workplace](https://handbook.mattermost.com/operations/workplace).
+6. @it33: Signs off on changes to signing authority ([example](https://github.com/mattermost/mattermost-handbook/pull/60)).
+
+Each PR should be reviewed by at least one approved reviewer. A build check requiring at least one approved review prior to a merge is planned, similar to other Mattermost repositories.
+
+Below is a list of permissions handbook contributors have access to:
+
+1. @jasonblais, @justinegeffen, @amyblais: Write permissions to the repository.
+2. @rbradleyhaas, @michaelschiffmm: Write permissions to the repository, but are not expected to make changes without reviews outside of [business operations](https://handbook.mattermost.com/operations/business-operations), nor make changes to table of contents without reviews. 
+4. Staff contributors: Submits changes to handbook.mattermost.com via PRs. Have access to request reviews, add labels, submit PR reviews, and be requested reviews from.
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-update-handbook.md]:
```suggestion
4. Staff contributors: Submits changes to handbook.mattermost.com via PRs. Have access to request reviews, add labels, submit PR reviews, and be requested as reviewers.
```

**Review by justinegeffen — CHANGES_REQUESTED** [review]
I've reviewed the steps for a PR in the handbook and updated the process.

**Comment by jasonblais** [comment]
Thank you! @justinegeffen editor suggestions applied

**Review by jasonblais — COMMENTED** [review]
I will revisit CLA process with HR following feedback from Schiff, and proving the granular permissions is in progress with Jason Deland. Merging given editor feedback applied, and we can iterate process further

**Pull request merged**